opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/OpenSearchSource.java

```diff
@@ -273,10 +291,13 @@ public SourceResponse query(QueryRequest queryRequest) throws UnsupportedQueryEx
                 Metacard metacard = null;
                 List<Result> resultQueue = new ArrayList<Result>();
                 try {
-                    metacard = inputTransformer.transform(responseStream);
-                } catch (IOException e) {
-                    LOGGER.debug("Problem with transformation.", e);
-                } catch (CatalogTransformerException e) {
+                    final byte[] bytes = IOUtils.toByteArray(responseStream);
```

Comment:
That should also show up as an attack vector in the threat models, so having some bounded value is a good thing. The system properties seems to be a reasonable approach. Might do some grepping around to see if it can be shared between multiple instances of this type of scenario (metadata records).